test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ControllerActionInvokerTest.cs

```diff
@@ -2357,6 +2357,60 @@ public class ControllerActionInvokerTest
         }
 
         [Fact]
+        public async Task InvokeAction_AsyncActionWithTaskOfObjectReturnType_AndReturningTaskOfActionResult()
+        {
+            // Arrange
+            var actionParameters = new Dictionary<string, object> { ["value"] = 3 };
+            IActionResult result = null;
+
+            var filter = new Mock<IActionFilter>(MockBehavior.Strict);
+            filter.Setup(f => f.OnActionExecuting(It.IsAny<ActionExecutingContext>())).Verifiable();
+            filter
+                .Setup(f => f.OnActionExecuted(It.IsAny<ActionExecutedContext>()))
+                .Callback<ActionExecutedContext>(c => result = c.Result)
+                .Verifiable();
+
+            var invoker = CreateInvoker(
+                new[] { filter.Object },
+                nameof(TestController.AsyncActionMethodReturningActionResultWithTaskOfObjectAsReturnType),
+                actionParameters);
+
+            // Act
+            await invoker.InvokeAsync();
+
+            // Assert
+            var testResult = Assert.IsType<TestActionResult>(result);
+            Assert.Equal(3, testResult.Value);
+        }
+
+        [Fact]
+        public async Task InvokeAction_ActionWithObjectReturnType_AndReturningActionResult()
+        {
+            // Arrange
+            var actionParameters = new Dictionary<string, object> { ["value"] = 3 };
+            IActionResult result = null;
+
+            var filter = new Mock<IActionFilter>(MockBehavior.Strict);
+            filter.Setup(f => f.OnActionExecuting(It.IsAny<ActionExecutingContext>())).Verifiable();
+            filter
+                .Setup(f => f.OnActionExecuted(It.IsAny<ActionExecutedContext>()))
+                .Callback<ActionExecutedContext>(c => result = c.Result)
+                .Verifiable();
```

Comment:
You can get rid of `Verifiable()` here since you're not verifying the mocks are executed.